src/consumer/component/dependencies/dependency-resolver/dependencies-resolver.ts

```diff
@@ -55,7 +55,7 @@ export default class DependencyResolver {
   allDependencies: AllDependencies;
   allPackagesDependencies: AllPackagesDependencies;
   // @ts-ignore AUTO-ADDED-AFTER-MIGRATION-PLEASE-FIX!
-  issues: $PropertyType<Component, 'issues'>;
+  issues: any; //$PropertyType<Component, 'issues'>;
```

Comment:
@qballer  , in Typescript it's actually easier, it should be `Component['issues']`